lib/apiservers/portlayer/restapi/handlers/storage_handlers.go

```diff
@@ -273,37 +273,40 @@ func (handler *StorageHandlersImpl) RemoveVolume(storage.RemoveVolumeParams) mid
 	return storage.NewRemoveVolumeOK() //TODO: this is just a stub for now.
 }
 
-// convert an SPL Image to a swagger-defined Image
-func convertImage(image *spl.Image) *models.Image {
-	var parent, selfLink *string
+//VolumesList : Lists available volumes for use
+func (handler *StorageHandlersImpl) VolumesList(params storage.ListVolumesParams) middleware.Responder {
+	defer trace.End(trace.Begin("storage_handlers.VolumeList"))
```

Comment:
this comment is for every trace call; writing the function names like that causes our logs to stutter. Trace already dump the name of the function and it's package so either write something usefull there (like dump of params etc.) or just pass an empty string